fe/fe-core/src/main/java/org/apache/doris/cloud/persist/UpdateCloudReplicaInfo.java

```diff
@@ -54,6 +59,9 @@ public class UpdateCloudReplicaInfo implements Writable {
     @SerializedName(value = "beIds")
     private List<Long> beIds = new ArrayList<Long>();
 
+    @SerializedName(value = "rids", alternate = {"replicaIds"})
```

Comment:
`alternate` is not needed by new fields.

Reply:
fixed